apps/sudoku-frontend/src/app/pages/classic-game/classic-game.component.html

```diff
@@ -0,0 +1,32 @@
+<se-sudoku-header additionalClass='single-line-heading' pageHeading='Classic'></se-sudoku-header>
+<div class='main-content'>
+  <div class='classic-game-container'>
+    <form [formGroup]='gridForm' (ngSubmit)='saveGrid()'>
+      <div class='game-grid'>
+        <table class='grid-table'>
+          <tr class='grid-row' *ngFor='let row of sudokuGrid; let rowIdx = index'>
+            <!-- only add styling classes if the indices are at specific number and don't double border on edge -->
+            <td class='grid-cell' *ngFor='let cellValue of row; let colIdx = index'
+                [ngClass]="{'bottom-line-cell': (rowIdx + 1) % 3 === 0  && rowIdx < 8, 'right-line-cell': (colIdx + 1) % 3 === 0 && colIdx < 8}">
+              <span *ngIf='sudokuGrid[rowIdx][colIdx]!==0; else elseblock'>
+                  <input [formControlName]='"cell" + colIdx  + "" + rowIdx'
+                         (focusout)='focusOutFunction()'
+                         [(ngModel)]='sudokuGrid[rowIdx][colIdx]'>
```

Comment:
formControlName und [(ngModel)] ned gemeinsam benutzen. Is entweder oder.

der state von dem input wird eh über formControlName in der FormGroup gspeichert.

Kannst ja dann `formGroup.get('name').value` machen

Reply:
habs refactored, merci für den hinweis, ging mir dabei nur um die default values, setze die jetzt aber eh in der component selbst, find ich schöner